### PR TITLE
support string input in serialize_object - #254

### PR DESCRIPTION
**Pull request by daonb**

### CODE REVIEW

File: src/zeep/helpers.py [hunk 1]
```diff
@@ -10,6 +10,9 @@ def serialize_object(obj):
     if obj is None:
         return obj
 
+    if isinstance(obj, basestring):
```
Comment (mvantellingen) [inline src/zeep/helpers.py]: six.text_type :-)

**Comment by daonb** [comment]
Thanks @dshingarev . I've approved my PR based on the @mvantellingen comments and `six`

**Comment by dshingarev** [comment]
Sounds good, thanks again.

**Comment by mvantellingen** [comment]
I've just refactored the serializer so that this should work. Let me know otherwise

**Pull request closed without merging**